### ISSUE TITLE
[Active] Pandora's Chest

### ISSUE THREAD

**Issue by ProbablyTez** [ProbablyTez/Agony#162]
- Twelve room charge
 - Has five chains, on use, removes a chain
 - If no chain remain, spawn 5 random items and destroy Pandora's Chest
 - Treasure and Devil pool

**Comment by NotNeal** [comment]
Not quite sure the payout is really worthwhile,
considering that you would need to clear 60 rooms to get the reward, if you get this on the first or second floor, you would probably be in the womb by the time you could open the chest, by then 5 items would be a bit of a boon, but it wouldn't exactly have been worth potentially passing up a good active item. I do like the concept tho, just make the payout better.

**Comment by dedChar** [comment]
What if we would set the room charge to 6? I think you have a point there @NotNeal, but also remember there are batteries throughout the run that would accelerate the opening process :)
I think a 6 room charge would be good. Maybe we could even tie the payout amount to the current floor? I dunno.

**Comment by ProbablyTez** [comment]
Yeah I wasn't sure about the math so the numbers are placeholders. I do feel like 6 charges is too few though, too bad we can't have 8 room charge items.

**Comment by dedChar** [comment]
Mechanic rework:   
- 6 Room Charge
- If it was used at least once and then dropped, it will reappear again on item pedestals
- Curse/Secret/Error room pool
- First Use:
    - All stats up
- Second Use:
    - Bad: Curse on every floor
    - Good: Spawns two items of current itempool
- Third Use:
    - Bad: Devil/Angel Deals are locked
    - Good: Small damage up (1.5)
- Fourth Use:
    - Bad: Spawn enemies in current room???
    - Good: ????
- Fifth (final) Use:
    - Bad: ???
    - Good: ??? 
- All effects are permanent
